src/emscripten-optimizer/colors.h

```diff
@@ -17,6 +17,7 @@
 #ifndef wasm_color_h
 #define wasm_color_h
 
+#ifndef WIN32
 #include <unistd.h>
```

Comment:
Not quite, that gives me errors for `isatty` and `STDOUT_FILENO`. I could certainly put another `#ifdef` in `use` though.